C++/C.sublime-syntax

```diff
@@ -88,7 +88,7 @@ contexts:
         - include: string_escaped_char
 
   string_escaped_char:
-    - match: '\\(\\|[abefnprtv''"?]|[0-3]\d{,2}|[4-7]\d?|x[a-fA-F0-9]{,2}|u[a-fA-F0-9]{,4}|U[a-fA-F0-9]{,8})'
+    - match: \\(?:\\|[abefnrtv\'"?]|[0-7]{1,3}|x[\da-fA-F]+|u[\da-fA-F]{4}|U[\da-fA-F]{8})
```

Comment:
Since the regex engine used by Sublime Text is unicode aware, `\d` will match the decimal digits on http://www.fileformat.info/info/unicode/category/Nd/list.htm. Hence, in this situation I believe `0-9` is more correct.